src/go/k8s/internal/controller/test/suite_test.go

```diff
@@ -203,8 +203,6 @@ var _ = BeforeSuite(func(suiteCtx SpecContext) {
 	// Helm Release Controller
 	helmRelease := helmController.HelmReleaseReconcilerFactory{
 		Client:        k8sManager.GetClient(),
-		Config:        k8sManager.GetConfig(),
-		Scheme:        k8sManager.GetScheme(),
 		EventRecorder: k8sManager.GetEventRecorderFor("HelmReleaseReconciler"),
 	}
```

Comment:
How this can work without GetClusterConfig and FieldManager?

https://github.com/redpanda-data/redpanda-operator/pull/43/files#diff-4ae1606f36f5fdbfa83dff5c37e5469ce0dd672e3fa7cc06c7ddfed384a19f91R163-R166